BUILD

```diff
@@ -1111,3 +1178,75 @@ sh_test(
         "update_file_lists.sh",
     ],
 )
+
+java_proto_library(
+    name = "test_messages_proto2_java_proto",
+    deps = [":test_messages_proto2_proto"],
+)
+
+java_proto_library(
+    name = "test_messages_proto3_java_proto",
+    deps = [":test_messages_proto3_proto"],
+)
+
+java_proto_library(
+  name = "conformance_java_proto",
+  deps = [":conformance_proto"],
+)
+
+java_lite_proto_library(
+    name = "test_messages_proto2_java_proto_lite",
+    deps = [":test_messages_proto2_proto"],
+)
+
+java_lite_proto_library(
+  name = "conformance_java_proto_lite",
+  deps = [":conformance_proto"],
+)
+
+java_lite_proto_library(
+    name = "test_messages_proto3_java_proto_lite",
+    deps = [":test_messages_proto3_proto"],
+)
+
+java_binary(
+  name = "conformance_java",
```

Comment:
It feels like the conformance stuff should go in a separate `conformance/BUILD` file. I guess we have some conformance stuff already in the main BUILD file, so maybe moving it all in one go should be a follow-up CL.

Reply:
Yeah, I was hoping to rectify the grossness in a followup, right now I am just augmenting what we have to at least be consistent for future refactors. I am not sure why this is not in its own folder.